platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -827,6 +827,8 @@ def get_transceiver_info(self):
                 for key in qsfp_compliance_code_tup:
                     if key in sfp_interface_bulk_data['data']['Specification compliance']['value']:
                         compliance_code_dict[key] = sfp_interface_bulk_data['data']['Specification compliance']['value'][key]['value']
+                if sfp_interface_bulk_data['data']['Extended Specification compliance']['value'] != "Unspecified":
```

Comment:
can you please add comment why it is needed and what is the motiviation of this code change?

Reply:
sometimes the cable can support additional 'Specification Compliance'.
we should parse and display this additional support from the EEPROM to get the 'Extended Specification Compliance' field.